website/content/docs/configuration/kms-library.mdx

```diff
@@ -2,8 +2,8 @@
 layout: docs
 page_title: Kms Library - Configuration
```

Comment:
Looking at other files - should this be something like Configure KMS library? At a minimum KMS should be all uppercase I think, and made to match the h1?

Reply:
Actually, it should be "kms_library - Configuration" since it's an actual stanza